account_invoice_pricelist/models/account_move.py

```diff
@@ -39,6 +40,7 @@ def _compute_pricelist_id(self):
                 invoice.partner_id
                 and invoice.is_sale_document()
                 and invoice.partner_id.property_product_pricelist
+                and not invoice.pricelist_id
```

Comment:
This changes the old behavior, where changing the customer/company of an invoice triggered an update of the pricelist, and it doesn't make sense: why would we want to keep the old pricelist if the customer/company changed?

I opened https://github.com/OCA/account-invoicing/pull/2085 to revert this change and only introduce the `readonly=False` attribute for the field.

Reply:
@jdidderen I tested it locally and I couldn't replicate the bug:
- I created a SO for a given customer
- I changed the pricelist to one that's not the customer's default pricelist
- I created an invoice from the SO
- the invoice had the correct pricelist (the one from the SO)

Care to explain how to reproduce the bug?

Reply:
@jdidderen Would be great if you can add a test (more likely in https://github.com/OCA/account-invoicing/pull/1959) that would highlight the issue you mention